cosmosdb/client_endpoint.bal

```diff
@@ -267,12 +276,13 @@ public client class DataPlaneClient {
         return mapJsonToStoredProcedure(jsonResponse);
     }
 
-    # List information of all stored procedures.
+    # Lists information of all stored procedures.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container which contains the stored procedures
-    # + resourceReadOptions - The `ResourceReadOptions` which can be used to add additional capabilities to the request
-    # + return - If successful, returns a `stream<StoredProcedure, error>`. Else returns `Error`.
+    # + resourceReadOptions - The `cosmos_db:ResourceReadOptions` which can be used to add additional capabilities to 
```

Comment:
Update this to be clear on what `cosmos_db:ResourceReadOptions` is, and replace "which" with "that".